src/middlewared/middlewared/schema.py

```diff
@@ -288,6 +303,41 @@ def validate(self, value):
         return super().validate(value)
 
 
+class Time(Str):
```

Comment:
We have a Time in validators already.

You need this to return the time object? Is there a sense to keeping Time validator around if this exists?

Reply:
Yes, sometimes we need to validate Time string and sometimes we need to parse Time into object, so I think we should have both